openwisp_notifications/tests/test_notifications.py

```diff
@@ -851,3 +852,28 @@ def test_notification_cache_update(self):
         self.assertEqual(notification.target.username, 'new operator name')
         # Done for populating cache
         self.assertEqual(operator_cache.username, 'new operator name')
+
+class test_email_notification(TestOrganizationMixin, TransactionTestCase):
+    def setUp(self):
+        self.user=self._create_user()
+        self.admin = self._create_admin()
+        self.target_obj = self._get_org_user()
+        self.data = dict(
+            email_subject='Test Email subject', url='https://localhost:8000/admin'
+        )
```

Comment:
We don't need to add a dedicated test class here. You can add this test to `TestTransactionNotifications` class. 

P.S.: The python convention is to use PascalCase for naming classes not snake_case. Refer https://namingconvention.org/python/